tests/report_tests.py

```diff
@@ -230,3 +230,31 @@ def test_report_type_exclude(self):
         results = self._results(api.url_for(ReportsView, committee_id=committee_id, report_type=['-M3']))
         self.assertTrue(all(each['report_type'] in ['Q2', 'TER'] for each in results))
 
+    def test_ie_only(self):
+        factories.ReportsIEOnlyFactory(
```

Comment:
What's the purpose of the first call to `ReportsIEOnlyFactory` and the `flush` afterwards?